### ISSUE TITLE
Logout Tweaks: Timer issue on potions

### ISSUE THREAD

**Issue by KidSith** [OrianaVenture/VentureValheim#37]
I had 38 seconds left on my frost resist when I decided to log off, now everytime I pop a frost resist it only goes up to 38 seconds

**Comment by buck2202** [comment]
I think this is an issue for rested as well. It seems to go to the time at log-in rather than the correct rested time for the comfort level. I logged in with 15 minutes, and when I rested at comfort 2, got 15 minutes again.

**Comment by OrianaVenture** [comment]
I have not been able to reproduce this issue. I suspect there may be a mod conflict. If both of you can share your mod lists that would be helpful.

**Comment by buck2202** [comment]
just 
```denikson-BepInExPack_Valheim-5.4.2202
BetterUI_ForeverMaintained-BetterUI_ForeverMaintained-2.5.6
Azumatt-AzuClock-1.0.3
Azumatt-Recycle_N_Reclaim-1.3.4
JereKuusela-Server_devcommands-1.81.0
VentureValheim-Venture_Logout_Tweaks-0.5.1
Advize-PlantEasily-1.8.0
```
for me

I tried to reproduce it again last night after my comment and it did not happen again, might not be a 100% thing. Will try to run with the log open and see if there are any hints in there if it happens again

**Comment by nbusseneau** [comment]
I have had this happening to others on the server and myself as well, both meads and rested buffs. I haven't found a way to reproduce it consistently yet.

**Comment by nbusseneau** [comment]
I don't actually know, I hadn't set up 0.5.0 in the modpack, we played straight with 0.5.1.

**Comment by OrianaVenture** [comment]
Found the issue. Code is modifying the original status effect rather than a copy, causing the values to be incorrect. Will be pushing out a fix in 0.5.2. Thank you for the reports!

**Comment by nbusseneau** [comment]
Very nice, thanks for the fix!
